kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -921,49 +942,64 @@ class ResidualBasedBlockBuilderAndSolver
     ///@}
     ///@name Protected member Variables
     ///@{
-    TSystemMatrixType mT;
-    TSystemVectorType mConstantVector;
-    std::vector<int> mSlaveIds;
-    std::vector<int> mMasterIds;
 
-   ///@}
+    TSystemMatrixType mT;              /// This is matrix containing the global relation for the constraints
+    TSystemVectorType mConstantVector; /// This is vector containing the rigid movement of the constraint
+    std::vector<IndexType> mSlaveIds;  /// The equation ids of the slaves

```

Comment:
are you still using mSlaveIds this for anything?

Reply:
Yep, but it is checked if inactive